netbox/users/models.py

```diff
@@ -227,6 +238,24 @@ def is_expired(self):
             return False
         return True
 
+    def validate_client_ip(self, raw_ip_address):
+        """
+        Checks that an ip address falls within the allowed ips.
+        """
+        if not self.allowed_ips:
+            return True
+
+        try:
+            ip_address = ipaddress.ip_address(raw_ip_address)
+        except ValueError:
+            raise ValidationError(f"{raw_ip_address} is an invalid IP address")
+
+        for ipnet in self.allowed_ips:
+            if ip_address in ipaddress.ip_network(ipnet):
```

Comment:
Try to avoid using overly terse variable names; they can impede comprehension.
```suggestion
        for ip_network in self.allowed_ips:
            if ip_address in ipaddress.ip_network(ip_network):
```